.env.example

```diff
@@ -13,4 +13,9 @@ DRIVE_DISK=fs
 APP_URL=http://localhost:3333
 APP_NAME=ToPWR_Backend
 APP_VERSION=1.0.0
-SESSION_DRIVER=cookie
\ No newline at end of file
+SESSION_DRIVER=cookie
+SMTP_HOST=host.pl
+SMTP_PORT=outgoing_port
+SMTP_USERNAME=mail@mail.pl
```

Comment:
really minor: use `example.org` instead, that domain exists for this exact purpose